kep/20200903-node-resources-allocatable-scoring.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Let's rename this file into `kep/48-node-resources-allocatable-scoring/README.md`, and move the metadata info (L1 to L15) to `kep/48-node-resources-allocatable-scoring/kep.yaml`.

The latest examples can be checked out under https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep.

Reply:
Done